packages/client/hmi-client/src/components/responsive-matrix/label-cols.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+	<div class="label-container col" :style="labelStyle">
+		<div class="label" v-for="(value, idx) in labelValues" :key="idx" :style="getLabelStyle(idx)">
+			{{ value }}
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { PropType } from 'vue';
+
+import { Point } from 'pixi.js';
```

Comment:
Would prefer to not have a coupling to `pixi` here.

Reply:
Forget this, since you are passing in a PIXI viewport anyway....didn't realize that. 
But since you are passing in a PIXI viewport you should be able to specify a type instead of leaving it as `null`

Reply:
yup added type to viewport